src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import React, { FC } from 'react';
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+}
+
+export const Pagination: FC<Props> = React.memo(
+  ({
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  }) => {
+    const countOfPages = Math.ceil(total / perPage);
+
+    const pageList = getNumbers(1, countOfPages);
+
+    const prevButtonIsDisabled = currentPage === 1;
+
+    const nextButtonIsDisabled = currentPage === countOfPages;
+
+    return (
+      <ul className="pagination">
+        <li className={classNames(
+          'page-item',
+          { disabled: prevButtonIsDisabled },
+        )}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={prevButtonIsDisabled}
+            onClick={() => {
+              if (currentPage !== 1) {
```

Comment:
```suggestion
              if (!prevButtonIsDisabled ) {
```
you can simplify this by using a created variable